provision.py

```diff
@@ -51,7 +51,7 @@
     sys.exit(1)
 
 # TODO: Parse arguments properly
-if "--travis" in sys.argv:
+if "--travis" in sys.argv or "--docker" in sys.argv:
```

Comment:
We'll probably want to switch to real argument parsing here, but that can be a separate patch we do later